projects/js-packages/webpack-config/src/webpack.js

```diff
@@ -30,8 +30,27 @@ const optimization = {
 	concatenateModules: false,
 	emitOnErrors: true,
 };
+
+const jsPackagesDirPath = path.join( __dirname, '../../' );
+// List of directory/package names inside `js-packages`
+const packageNames = fs
+	.readdirSync( jsPackagesDirPath )
+	.filter( item => fs.lstatSync( jsPackagesDirPath + item ).isDirectory() );
+
+/**
+ * In order to allow webpack resolve internal js-packages correctly,
+ * We need to create aliases for all of the packages.
+ */
+const alias = packageNames.reduce( ( prevValue, name ) => {
+	return {
+		...prevValue,
+		[ `@automattic/jetpack-${ name }` ]: path.join( jsPackagesDirPath, name ),
+	};
+}, {} );
+
 const resolve = {
 	extensions: [ '.js', '.jsx', '.ts', '.tsx', '...' ],
+	alias,
 };
```

Comment:
This is to fix `SassError: Can't find stylesheet to import.` error here

https://github.com/Automattic/jetpack/runs/5732604305?check_suite_focus=true

The reason for that being mentioned [here](https://github.com/webpack/webpack/discussions/14630#discussioncomment-1587496).

> We don't support alises from `tsconfig.json`, you should set `aliases` in `webpack.config.js` using the `resolve.alias` option

Reply:
We didn't have to do this before. What of your other changes made this suddenly necessary?

Reply:
Not sure about what caused. It happened after I rebased the branch.